frontend/app/dashboard/src/views/dashboard/DashboardMenu.vue

```diff
@@ -225,7 +234,7 @@ export default class Menu extends Mixins(NavigationMixin) {
         if (!didSet && this.enableMemberModule && parts.length >= 2 && parts[0] == "groups") {
             for (const group of this.organization.groups) {
                 if (parts[1] == Formatter.slug(group.settings.name)) {
-                    this.openGroup(group, false)
+                    this.openGroup(this.organization.categoryTree,group, false)
```

Comment:
I'm not sure what to do with this one because the categoryTree is not directly available here.

Reply:
There are some methods to get the parent category of a group, but theoretically a group can be in multiple categories (not yet possibel via UI, but possible). The best way would be to move the logic to the editing of a category (then you already have the right category + this is a very logical place to do so)